lib/board.rb

```diff
@@ -0,0 +1,23 @@
+class Board
+  @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+  class << self
+    attr_reader :board
+  end
+
+  def self.change_board(new_board)
+    @board = new_board
+  end
```

Comment:
Can't we achieve this with an attr_accessor???

Reply:
No I don't think so because we intend to use it for class global variable